src/antsibull_build/constants.py

```diff
@@ -33,6 +33,10 @@
 ANSIBLE_DOCUMENTATION_RANGES: dict[str, PypiVer] = {
     "2.13": PypiVer("2.13.11"),
     "2.14": PypiVer("2.14.8"),
+    # Temporarily use devel porting guide for 2.19.0a1
+    # so https://github.com/ansible/ansible-documentation/pull/2429
+    # gets included.
+    "2.19": PypiVer("2.19.0a2"),
```

Comment:
The ansible-core version is 2.19.0b1, not 2.19.0a1... :facepalm: I've corrected this in 46f36b8a52d4b4437861d3c963138fcc2d0c2314 and will do another release.